client/src/inside/components/discharge/ehr-summary-tableV2.js

```diff
@@ -38,9 +37,8 @@ Referrals: Pulled in from the referrals to other disciplines page.
     Referral time {fqn: referrals.table.appointmentTime}
  */
 
-const pDefs = new PeriodDefs()
-const medSchedules = pDefs.keyNameList
-// console.log('EhrSummaryHelp medSchedules', medSchedules)
+const medSchedules = [] // TODO get MARs
```

Comment:
Is this TODO still a todo?

Reply:
I don't think it is. I think that I added this in an earlier version and forgot to remove it... I will be removing it now, thank you!